dev-util/gammaray/gammaray-2.9.1.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="High-level runtime introspection tool for Qt applications"
+HOMEPAGE="https://github.com/KDAB/GammaRay"
+
+if [[ ${PV} == 9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/KDAB/GammaRay.git"
+else
+	SRC_URI="https://github.com/KDAB/GammaRay/releases/download/v${PV}/${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="GPL-2+"
+SLOT=0
+IUSE="3d bluetooth concurrent designer doc geolocation printsupport
+	script scxml svg test qml wayland webengine"
+
+RDEPEND="
+	dev-qt/qtcore
```

Comment:
Not quite, you still need to depend on `SLOT 5.

Reply:
Hmm, I suppose you are right. Removing Qt4 isn't enough and leaving Qt5 only isn't enough to be safe.